volatility/framework/plugins/windows/getsids.py

```diff
@@ -0,0 +1,257 @@
+# This file is Copyright 2019 Volatility Foundation and licensed under the Volatility Software License 1.0

```

Comment:
Usual changes please (2020 and don't include author information in individual files, happy for you to create an AUTHORS.txt or CREDITS.txt if you want)